### DISCUSSION TITLE
Local recording has no audio

### DISCUSSION THREAD

**Discussion by yasmagic** [mynaparrot/plugNmeet-server#610] [General]
The generated WEBM file from local recording has no audio.

**Comment by Saber-ebr** [comment]
I have the same issue.
Did you find any solutions?

**Comment by jibon57** [comment]
Let me clarify how local recording works, so we can reference this in the future.

This issue is quite common, as many users are not fully aware of how the local recording feature operates. To use local recording, you must use **Google Chrome** and log in with a different user account. Open Chrome in a new window (not just a new tab) and join the session as an administrator (not as a participant). Once in the session, start Local Recording. A new window will prompt you to share your current tab. At the bottom, be sure to enable the option to record the tab’s audio. This step is essential to capture the entire session audio. If these instructions aren’t followed, the audio will not be recorded, which is expected.

If you prefer an alternative approach, begin local recording after you have already shared your microphone. When you start recording, the system captures audio from your current audio devices. However, if you start recording before sharing your microphone, your microphone audio will be excluded—this is normal and there is no workaround. Note that local recording is entirely browser-based, and tab audio capture is only supported in **Google Chrome**. Attempting this in Firefox or any non-Chromium browser will not work as intended.
